components/automate-gateway/api/iam/v2beta/policy.proto

```diff
@@ -177,4 +177,12 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "system:iam:resetToV1";
     option (chef.automate.api.iam.policy).action = "system:iam:reset";
   };
+  rpc IntrospectAllProjects (ListProjectsReq) returns (ListProjectsResp) {
+    option (google.api.http).get = "/iam/v2beta/introspect_projects";
+    option (chef.automate.api.policy).resource = "auth_introspection:introspect_all_projects";
+    option (chef.automate.api.policy).action = "read";
+    option (chef.automate.api.iam.policy).resource = "iam:introspect";
+    option (chef.automate.api.iam.policy).action = "iam:projects:getAllProjects";
```

Comment:
Re-purposing what IntrospectAllProjects does--and at the same time putting it on the /iam/v2beta endpoint path, where it belongs. There is one anomaly here: Had to adjust the action to *not* be under `iam:introspect:*`.
Without that change IntrospectAllProjects returns *all* projects rather than all *allowed* projects (because the system policy for iam:introspect:* includes projects: `*`).